src/Compilers/CSharp/Test/Emit/BreakingChanges.cs

```diff
@@ -1006,7 +1006,8 @@ public class c
             comp.VerifyDiagnostics();
         }
 
-        [Fact, WorkItem(530518, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/530518")]
+        [ConditionalFact(typeof(DesktopOnly))]
```

Comment:
Are there any tests that will stay conditional in the mid- or long- term? If yes, it may be good to comment on why a test is desktop only, so that we can track which ones to re-enable once a certain issue or category of issues is resolved.

Reply:
This is coming in a later PR. After talking with Andy it seemed better to make our primitives simpler here, include a suppression reason + revisit all of the tests we've disabled . 

---
In reply to: [171413418](https://github.com/dotnet/roslyn/pull/25050#discussion_r171413418) [](ancestors = 171413418)